compiler/dyno/test/resolution/testOperatorOverloads.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* #include "chpl/parsing/parsing-queries.h" */
+/* #include "chpl/resolution/resolution-queries.h" */
+/* #include "chpl/types/ComplexType.h" */
+/* #include "chpl/types/IntType.h" */
+/* #include "chpl/types/Param.h" */
+/* #include "chpl/types/RealType.h" */
+/* #include "chpl/uast/Module.h" */
+#include "common.h"
+
+// always check assertions in this test
+#ifdef NDEBUG
+#undef NDEBUG
+#endif
+
+#include <cassert>
+
+static void test1() {
+  Context ctx;
+  auto context = &ctx;
+
+  // no declaration (cast should fail)
+  QualifiedType qt1 = resolveTypeOfXInit(context, "\
```

Comment:
Take a look at the fancy multi-line strings in other tests:
https://github.com/chapel-lang/chapel/blob/1fc2ec9529000ad8906a71307122dcf9afb6000e/compiler/dyno/test/resolution/testFieldAccess.cpp#L49-L55
Saves the work of having to backslash your newlines!

Reply:
Looks a lot nicer, I'll use that!